x-pack/plugins/ml/server/routes/job_service.ts

```diff
@@ -644,6 +673,9 @@ export function jobServiceRoutes(
       validate: {
         body: schema.object(topCategoriesSchema),
       },
+      options: {
+        tags: ['access:ml:canCreateJob'],
```

Comment:
Is this too restrictive? Would `canGetJobs` be better? Or is this only ever going to be used in the job creation workflow?